tests/contest/contest/src/utils/linux_resource_memory.rs

```diff
@@ -0,0 +1,36 @@
+use super::ContainerData;
+use anyhow::{bail, Result};
+use libcgroups::v1::util::get_memory_data;
+use oci_spec::runtime::Spec;
+
+pub(crate) fn validate_linux_resource_memory(spec: &Spec, data: ContainerData) -> Result<()> {
+    let expected_memory = spec
+        .linux()
+        .as_ref()
+        .unwrap()
+        .resources()
+        .as_ref()
+        .unwrap()
+        .memory()
+        .as_ref();
```

Comment:
I think we can do without the `as_ref` calls here, as spec is already a immutable ref? Please check once.

Reply:
Oh ,the linux() function call returns a &Option<Linux>, so as_ref() was needed to make it a Option<&Linux>